public/app/features/alerting/unified/insights/grafana/InstanceStatusScene.tsx

```diff
@@ -0,0 +1,45 @@
+import { PanelBuilders, SceneFlexItem, SceneQueryRunner, SceneTimeRange } from '@grafana/scenes';
+import { DataSourceRef } from '@grafana/schema';
+
+const STATUS_COLORS = {
+  alerting: 'red',
+  pending: 'yellow',
+  nodata: 'blue',
+  normal: 'green',
+  error: 'orange',
+};
+
+export function getInstanceStatByStatusScene(
+  timeRange: SceneTimeRange,
+  datasource: DataSourceRef,
+  panelTitle: string,
+  status: 'alerting' | 'pending' | 'nodata' | 'normal' | 'error'
+) {
+  const query = new SceneQueryRunner({
+    datasource,
+    queries: [
+      {
+        refId: 'A',
+        instant: true,
+        expr: `sum by (state) (grafanacloud_grafana_instance_alerting_alerts{state="${status}"})`,
+      },
+    ],
+    $timeRange: timeRange,
+  });
+
+  return new SceneFlexItem({
+    height: '100%',
+    body: PanelBuilders.stat()
+      .setTitle(panelTitle)
+      .setDescription(panelTitle)
+      .setData(query)
+      .setOverrides((b) =>
+        b.matchFieldsWithName(status).overrideColor({
+          mode: 'fixed',
+          fixedColor: STATUS_COLORS[status],
```

Comment:
nit: we could use this `STATUS_COLORS` constant in the rest of the code we are setting the `fixedColor`, OR even better ,we could create a method to get this `overrideColor` param that I see it's being repeated in more places several times (using also this constant in it)